### PR TITLE
LPD-37329 Migrate NumericInput.testcase

### PR DESCRIPTION
**Pull request by veroglez**

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by veroglez** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:sf

**Comment by veroglez** [comment]
ci:test:sf

File: modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts [hunk 1]
```diff
@@ -132,6 +133,105 @@ test.describe('Form Configuration', () => {
...
+
+		// Select Numeric input
+
+		const numericInputId = await pageEditorPage.getFragmentId('Numeric');
+
+		await pageEditorPage.selectFragment(numericInputId);
+
+		// Check Mark as Required field
+
+		await page.getByLabel('Mark as Required', {exact: true}).check();
```
Comment (sandrodw3) [inline modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts:172]: You can do this with `changeFragmentConfiguration` and you don't have to select the fragment manually 😇

**Review by sandrodw3 — COMMENTED** [review]
LGTM! 🥰 just a super minor suggestion

File: modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts (hunk 1 above)
Comment (veroglez) [reply modules/test/playwright/tests/layout-content-page-editor-web/formContainer.spec.ts]: True!

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by veroglez** [comment]
ci:test:page-management-playwright

**Comment by sandrodw3** [comment]
ci:forward:force

**Pull request closed without merging**